Resources/Private/Javascript/Components/CartItems.js

```diff
@@ -24,27 +25,31 @@ const CartItems = () => {
   },[])
 
   return (
-    <div>
+    <div className='c-cart-items'>
       <Overlay loading={loading} />
 
       {cartItems.length > 0 ?
-        <ul>
+        <ul className='c-cart-items__list'>
           {cartItems.map(product => (
-            <li key={product.id}>
-              <img src={product.cover.url} />
-              <div>{product.label}</div>
-              <div>{product.quantity}</div>
-              <div>{formatCurrency(product.price.unitPrice)}</div>
-              <div>{formatCurrency(product.price.totalPrice)}</div>
+            <li className='c-cart-items__item' key={product.id}>
+              <img className='c-cart-items__image' src={product.cover.url} alt='' />
+              <div className='c-cart-items__label'>{product.label}</div>
+              <div className='c-cart-items__quantity'>{product.quantity}</div>
+              <div className='c-cart-items__unit-price'>{formatCurrency(product.price.unitPrice)}</div>
+              <div className='c-cart-items__total-price'>{formatCurrency(product.price.totalPrice)}</div>
             </li>
           ))}
         </ul>
       : 
         !loading ?
```

Comment:
This is a bit hard to read, why do we need another checked, if it is loaded here? As long as there is a loading overlay on top, the user knows whats happening

Reply:
I prefer showing some initial state (like this message) instead of just a loading-spinner, but that's up to you